src/views/submission-summary.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "layout.njk" %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+
+{% block pageTitle %}
+    Check your answers before submitting your appeal
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">
+                Check your appeal
+            </h1>
+
+            <h2 class="govuk-heading-m">Penalty details</h2>
+            <dl class="app-check-your-answers app-check-your-answers--short govuk-!-margin-bottom-8">
+                <div class="app-check-your-answers__contents">
+                    <dt class="app-check-your-answers__question">
+                        Company Number
+                    </dt>
+                    <dd class="app-check-your-answers__answer">
+                        {{ session.getExtraData('appeals').penaltyIdentifier.companyNumber }}
```

Comment:
can we variables for `session.getExtraData('appeals')` and  `session.getValue('signin_info')`?  We may want to think about what happens if this data doesn't exist

Reply:
I tried doing that on the controller side, but wasn't sure how to access the getExtraData and the getValue from the controller